src/test/tests/databases/blueprint.py

```diff
@@ -114,6 +122,21 @@ def test_mfem(tag_name, example_name, protocol):
 
     CloseDatabase(dbfile)
 
+def test_paren_vars():
+    TestSection("Variables With Parens")
+
+    #
+    # Testing bugfix for issue #4882.
+    #
+    OpenDatabase(uniform_root)
+    AddPlot("Pseudocolor", "mesh_topo/rho.Y_lp_CH2O_rp_")
```

Comment:
So, this is a little funky in that this test is using *internal* knowledge of VisIt's name mangling. It suggests to me that part of the story in #4908 is knowing where in the interface to map to/from the user's knowledge of the variable's names to VisIt's and vice-veresa. I mean, had we implemented the mangling correctly in the past, we would not have exposed users to having to include those mangled names in their python code. This isn't a comment about something to be corrected in this PR...just more food for thought for #4908.